.golangci.yml

```diff
@@ -13,6 +13,18 @@ issues:
     text: "SA1019: codec.BasicHandle is deprecated"
   # This is set to false to disregard the default EXC0011 exclusion
   # which shadows all the ST* checks. Refer https://github.com/golangci/golangci-lint/issues/2281
+  - path: "pkg/client/(.+)\\.go"
+    linters:
+      - importas
+  - path: "pkg/database/cosmosdb/zz_generated_(.+)\\.go"
+    linters:
+      - importas
+  - path: "pkg/operator/(apis|clientset|mocks)/(.+)\\.go"
+    linters:
+      - importas
```

Comment:
I understand `clientset` and `mocks`, but not `apis`. What's the reason we don't want those linted?

Reply:
I just reflected here https://github.com/Azure/ARO-RP/blob/be4bae3d0ce01c2c9ba3cc7fe7bf30b0a1cd78c2/hack/validate-imports/validate-imports.go#L155 .
but you are true.
After removing it, it didn't output error.
I'll fix this.